dev/MRTCore/packaging/MrtCore.targets

```diff
@@ -4,6 +4,8 @@
   <!-- We want MRT Core to NOT be used in WAP or UWP projects. Instead, we want the in-box PRI generation tooling to be used. -->
   <PropertyGroup>
     <EnableCoreMrtTooling Condition="'$(EnableCoreMrtTooling)'=='' and '$(WindowsAppContainer)'!='true' and '$(MSBuildProjectExtension)' != '.wapproj'">true</EnableCoreMrtTooling>
+    <!-- If MSBuildRuntimeType is Core then we are using dotnet build because it will fail since ExpandPriContent is not available in the dotnet enviroment -->
+    <EnableCoreMrtTooling Condition="'$(MSBuildRuntimeType)'=='Core'">false</EnableCoreMrtTooling>
```

Comment:
Should this be combined into the prior line? Extend the list of `and` values.

Feels like "enable this toolset" makes more sense based on the content of the project.

Can someone build a C# WinUI3 app with resources in `dotnet build`? Should they be able to?

Reply:
Should this be combined into the prior line? Extend the list of and values.
We could but I was worried the line will become too long. 

Can someone build a C# WinUI3 app with resources in dotnet build? Should they be able to?
I don't know if this is a valid scenario. I presume we would only build with dotnet build if we are on a dotnet app?
@DrusTheAxe @jamespik @MikeHillberg What do you all think?

Reply:
Can - today? I thought there were issues with `dotnet build` and work needed to make it work. @Scottj1s likely knows more but not currently available.

>I presume we would only build with dotnet build if we are on a dotnet app?

dotnet build and msbuild equivalent-ish, but that -ish can be significant in syntax, functionality and environment factors. There are differences, both obvious and subtle. Search for "dotnet build vs msbuild" for some of the details.

It's desirable to fully support `dotnet build` but it always sounded like feature-level work (not a simple bugfix).